### DISCUSSION TITLE
危急時刻提供可靠幫助，Google 如何運用 AI 提升搜尋安全性？

### DISCUSSION THREAD

**Discussion by timchen1216** [1101-GS3073-C/Python-Community-GS3073-C#353] [科技新知]
在危急時刻除了尋求他人協助以外，人們也常使用 Google 搜尋求助，比方說有人在 Google 上搜尋關於自殺、家暴的資訊時，就會看到全年無休的協談或是救助熱線。對 Google 而言，不是每一次都能知道人們究竟需要什麼樣的幫助，同時也希望做更多的事來引導人們獲得需要的資訊。為此，Google 搜尋研究員暨副總裁 Pandu Nayak 撰文說明，Google 搜尋引擎如何利用 AI 人工智慧模型 MUM，在關鍵時刻提供值得信賴的實用資訊。

具體來說，Google 透過在 Google I/O 2021 開發者大會就已亮相的 MUM（Multitask Unified Model），能夠自動、準確偵測危急狀況的搜尋，而且範圍比過去更廣。MUM 可以更清楚地釐清人們問題背後的意圖，以偵測使用者何時需要協助，這有助於 Google 在正確的時間點，提供可靠、可執行的資訊。

比方說，使用者在 Google 搜尋「自殺最常見的方式」或「自殺熱門地點」等等，系統過去可能會理解成一般資訊的搜尋，又或者針對搜尋字詞中的「地點」來提供資訊，而忽略「自殺」代表的涵意；如今則會顯示能夠提供幫助的具體資訊，例如諮詢機構的協談專線或網站。

![image](https://user-images.githubusercontent.com/95116299/172054064-8991fd0b-d024-4ec6-a998-c767c7fd7a36.png)
不過，若要確保使用者能夠安全地使用 Google 搜尋功能，就要協助他避開非預期的駭人內容，而這非常具有挑戰性。

Google 處理這種狀況的方法，有一種是透過安全搜尋功能，讓使用者能選擇是否要過濾掉煽情露骨的搜尋結果，這項功能會為未滿 18 歲的 Google 用戶預設開啟。就算沒有開啟安全搜尋功能，只要不是在尋找色情內容，Google 搜尋的系統仍會減少個人所不想要的色情搜尋結果。

對此 Google 還是有些改善空間，像是也使用 [BERT](https://technews.tw/2019/11/01/google-search-bert-algorithm/)（Bidirectional Encoder Representations from Transformers）這樣的進階 AI 技術，更加了解使用者在找什麼。BERT 幫助 Google 理解使用者是否真的在搜尋露骨內容，並大幅減少在搜尋結果中看到非預期內容的機會。其實光在 2021 年，BERT 的改進已經把那些非預期的駭人搜尋結果降低 30%，尤其是在搜尋種族、性傾向、性別相關內容時，減少煽情露骨的內容特別有效。

Google 運用 75 種語言來訓練 MUM，所以它能夠在這些語言中傳遞資訊，這也使 Google 能夠更有效地把保護的規模擴大到全世界。Google 採用 AI 技術協助減少搜尋結果中無用、甚至是危險的垃圾內容頁面；在接下來的幾個月，會用 MUM 來提升阻擋垃圾訊息的效能，並且擴展到訓練資訊不太多的語言裡。未來在全世界，Google 將能偵測到危急狀況的搜尋，並且和當地值得信賴的夥伴合作，以提供具有實用價值的資訊。

資料來源:https://technews.tw/2022/04/01/using-ai-to-keep-google-search-safe/

竟然是bert!!!那個用tpu也要訓練九天的超大型model

**Comment by lx3211** [comment]
雖然一心求死的人還是無法避免掉，但可以阻止那些還在猶豫的人

**Comment by 32844583** [comment]
MUM 可以理解人類言詞背後的意義，這跟ChatGPT應用到的模型與技術應該大同小異吧!

**Comment by brian431** [comment]
這幾個皆是大型自然語言模型如何派上用場的幾個好例子，在將來若是模型的成本更低，人們在使用搜尋引擎時將會看到更多有用且方便的功能。

**Comment by NcuJerryKao** [comment]
這讓我想到韓國的那橋，雖寫上溫標語 但反而造成反效果喔
https://www.storm.mg/lifestyle/67902

**Comment by BingRong1217** [comment]
感覺可以有效阻止一些有負面想法的人

**Comment by limitx0** [comment]
有感覺Google的搜尋結果越來越人性化和方便

**Comment by awehao** [comment]
感覺雖然是新突破，但僅僅只是一串字，很難讓有心人真正感受到幫助

**Comment by Chenszuhong** [comment]
這是個不錯的小麥 但是有時候收尋一些相關的資訊 就會跳出這個 有點影響使用

**Comment by larry40305** [comment]
本人也是經過了深思熟慮，在每個日日夜夜思考這個問題。搜尋安全性對我來說有著舉足輕重的地位，必須要嚴肅認真的看待。

**Comment by boson13579** [comment]
藉由訓練MUM，Google能即時阻止並提供有負面想法的人紓解情緒的管道，挽救更多遊走邊緣的生命。
